### DISCUSSION TITLE
Any recommendations for class name order?

### DISCUSSION THREAD

**Discussion by creart-jetsai** [master-co/css#45] [Question]
Do you or would you like to provide docs related to Master CSS class name ordering.

It would be nice if you plan to release a best practice page!

Btw, Tailwind CSS has their [recommendations](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier#how-classes-are-sorted) and their prettier [sorter](https://github.com/tailwindlabs/prettier-plugin-tailwindcss), but it seems a heavy work. Docs are enough.

**Comment by 1aron** [comment]
@creart-jetsai Not yet, but I think it's a necessary feature. Great proposal 👍🏻

**Comment by 1aron** [comment]
One year later 🚀 https://beta.css.master.co/docs/code-linting
